### PR TITLE
Update help and error message of ares-pacakge

### PR DESCRIPTION
**Pull request by ssuminahn**
:Release Notes:
Update help and error message of ares-pacakge

:Detailed Notes:
Update help and error message of ares-pacakge
for info and info-detail option

:Testing Performed:
1. Pass unit test
2. Pass eslint
4. Check help updated and error message

:Issues Addressed:
[WRN-12303] Write ipk analyze document

### CODE REVIEW

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

File: lib/base/error-handler.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ const log = require('npmlog'),
             "NOT_SUPPORT_RUNNINGLIST" : "Not supported method to get running app information",
             "NOT_SUPPORT_JOURNALD" : "journald does not support the option",
             "NOT_SUPPORT_PMLOGD" : "pmlogd does not support the option",
-            "NOT_SUPPORT_XZ" : "Not Supported .xz extension. Analyze only ipk packaged with CLI",
+            "NOT_SUPPORT_XZ" : "xz file is not supported. Only .ipk file packaged with the CLI is supported.",
```
Comment (YoungeunKim) [inline lib/base/error-handler.js:107]: Please remove "." end of sentence.

**Comment by YoungeunKim** [comment]
Code 👎

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

**Review by YoungeunKim — APPROVED** [review]
Code 👍 
Verify 👍

**Pull request merged**